examples/rules_cc/WORKSPACE

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Uber Technologies, Inc.
+# Licensed under the MIT License
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+HERMETIC_CC_TOOLCHAIN_VERSION = "v2.0.0-rc1"
+
+http_archive(
+    name = "hermetic_cc_toolchain",
+    sha256 = "43a1b398f08109c4f03b9ba2b3914bd43d1fec0425f71b71f802bf3f78cee0c2",
+    urls = [
+        "https://mirror.bazel.build/github.com/uber/hermetic_cc_toolchain/releases/download/{0}/hermetic_cc_toolchain-{0}.tar.gz".format(HERMETIC_CC_TOOLCHAIN_VERSION),
+        "https://github.com/uber/hermetic_cc_toolchain/releases/download/{0}/hermetic_cc_toolchain-{0}.tar.gz".format(HERMETIC_CC_TOOLCHAIN_VERSION),
+    ],
+)
+
+load("@hermetic_cc_toolchain//toolchain:defs.bzl", zig_toolchains = "toolchains")
+
+zig_toolchains()
+
+register_toolchains(
+    "@zig_sdk//toolchain:linux_amd64_gnu.2.19",
```

Comment:
```suggestion
    "@zig_sdk//toolchain:linux_amd64_gnu.2.28",
```

Reply:
Let's use an up-to-date libc in examples. Users can downgrade if they need it themselves.